src/index.js

```diff
@@ -50,17 +50,17 @@ module.exports = {
             let babelOptions = transformConfig.babelOptions;
 
             let rootPackage = lassoPackageRoot.getRootPackage(path.dirname(filename));
-
+            let curDir = process.cwd();
```

Comment:
Is there a reason why you're using `process.cwd()` instead of using `path.dirname(filename)`? I think that will work fine for your use case and then line 63 wouldn't have to be set at all.

If that is true, you can also pull the `path.relative(curDir, filename)` outside of the `if/else` to be shared. It would become:

```js
let curDir = path.dirname(filename);
const rootPackage = lassoPackageRoot.getRootPackage(curDir);

if (babelOptions) {
  ...
} else {
  ...
  if (!babelOptions) {
    // No babel config... Don't do anything
    return code;
  }

  babelOptions.babelrc = false;
}

babelOptions.filename = path.relative(curDir, filename);
let babel = getBabel();
let result = babel.transform(code, babelOptions);
return result.code;
```

Reply:
yeah. that makes sense. Will push this change.